charactersheet/charactersheet/templates/spells.tmpl.html

```diff
@@ -52,8 +52,15 @@
           <!-- /ko -->
           <td data-bind="text: spellName, click: $parent.editSpell"
                href="#"></td>
+
+          <!-- ko if: spellLevel() != 0 -->
```

Comment:
Thanks for taking this task on, @jkrooskos, since the 'level 0' thing does cause some confusion. 
Typically, we don't want logic, like the ko ifs, in the view. This leads to inconsistency and confusion when debugging. 
We have started to use the <something>Label paradigm to display information to the view. I would suggest that the spells view model have a new variable called spellLevelLabel, which will incorporate the existing logic here, and will simply return a string. Finally, the view will change to data-bind='text:spellLevelLabel'.